SpatialGDK/Source/SpatialGDK/Public/Interop/RPCs/RPCQueues.h

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/RPCs/RPCService.h"
+
+namespace SpatialGDK
+{
+template <typename Payload, typename AdditionalSendingData = EmptyData>
+struct TRPCLocalOverflowQueue : public TRPCQueue<Payload, AdditionalSendingData>
+{
+	TRPCLocalOverflowQueue(FName InName, TRPCBufferSender<Payload>& Sender)
+		: TRPCQueue<Payload, AdditionalSendingData>(InName, Sender)
+	{
+	}
+
+	void FlushAll(RPCWritingContext& Ctx, const typename TWrappedRPCQueue<AdditionalSendingData>::SentRPCCallback&) override;
```

Comment:
FlushAll/Flush/FlushQueue are very similar sounding, could you distinguish better their intended purposes?